docs/view_components/view_component_template.rst

```diff
@@ -1,25 +1,72 @@
-.. reference for this view-component
-.. you can refer to this component using :ref:`<component_name>-label`
+.. Refer to a component using :ref:`<component_name>-label`
 
-.. _<component_name>-label:
+.. _view_component_template-label:
 
+Template for view components
+----------------------------
+.. Adapt tp your view component from here
+
+------------------------------------------------------
```

Comment:
What is the use of this paragraph? One should erase it when overtaking the template?

Reply:
This paragraph is here because you wanted the template to appear in tool interface
I can adapt that